### PR TITLE
adding fixture test for #s + updating callback on voice

### PR DESCRIPTION
**Pull request by chelpu**

### CODE REVIEW

File: numbers/response.go [hunk 1]
```diff
@@ -5,12 +5,18 @@ import (
 	"regexp"
 )
 
+type capabilities struct {
+	Voice bool `json:"voice,string"`
```
Comment (natebrennand) [inline numbers/response.go]: these are plain booleans in the fixture string you have, do we need to put `string` in the json annotation?

File: voice/callback.go [hunk 2]
```diff
@@ -71,6 +71,8 @@ func CallbackHandler(callbackChan chan Callback) http.HandlerFunc {
 			return
 		}
 		resp.WriteHeader(http.StatusOK)
-		callbackChan <- cb
+		go func() {
```
Comment (natebrennand) [inline voice/callback.go:74]: :fire: hott :fire:

Comment (chelpu) [inline voice/callback.go:74]: ikr :hand:

**Comment by natebrennand** [comment]
:+1:

**Pull request merged**